Homework_2/hw_task_1.py

```diff
@@ -0,0 +1,18 @@
+"""Напишите программу, которая получает целое число и
+возвращает его шестнадцатеричное строковое представление.
+Функцию hex используйте для проверки своего результата."""
+
+HEX = 16
+DCT = {10: 'A', 11: 'B', 12: 'C', 13: 'D', 14: 'E', 15: 'F'}
+
+n = int(input('Введите число: '))
+num = n
+lst = []
+while num:
+    num, rest = divmod(num, HEX)
+    if rest < 10:
+        lst.append(str(rest))
+    else:
+        lst.append(DCT[rest])
```

Comment:
Если расширить словарь `DCT` по остальным цифрам системы, то этот код можно упростить
```suggestion
    lst.append(DCT[rest])
```